README.md

```diff
@@ -1,26 +1,36 @@
-Bitcoin Core integration/staging tree
+Namecoin Core integration/staging tree
 =====================================
 
-[![Build Status](https://travis-ci.org/bitcoin/bitcoin.svg?branch=master)](https://travis-ci.org/bitcoin/bitcoin)
```

Comment:
I think build status should stay.  Replacing bitcoin/bitcoin with namecoin/namecore should yield the correct Travis CI URL's.